### PR TITLE
chore: Add issue and PR templates

### PR DESCRIPTION
**Pull request by carpusherw**
# Add issue and PR templates

## Change Summary

- Add issue templates of
  - Bug report
  - Feature request
  - New plugin request
- Add an PR template

## PR Checklist

- [x] I've read and followed the [Contribution Guide](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/.github/CONTRIBUTING.md).
- Documents/Readmes
    - [x] Updated accordingly
    - [ ] Not required

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+-->
+
+# <PATH_TO_THE_PLUGIN>
+<!-- For example: # post-scan-actions/aws-python-promote-or-quarantine -->
+
+## Describe the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
```
Comment (Laurence-Cheng) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: You don't start every words with capital character for other titles.
So my suggestion is only for format consistency.

```suggestion
## To reproduce
```
or
```suggestion
## How to reproduce
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+-->
+
+## Problem to solve
+
+<!--
+A clear and concise description of what the problem is.
+Ex. I'm always frustrated when [...]
+-->
+
+## Proposed Solution
```
Comment (Laurence-Cheng) [inline .github/ISSUE_TEMPLATE/feature_request.md:28]:
```suggestion
## Proposed solution
```

**Review by dougc-trend — DISMISSED** [review]
Looks good

**Review by Laurence-Cheng — APPROVED** [review]
LGTM

**Review by pax-chiu — APPROVED** [review]

**Pull request merged**